explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -314,9 +312,10 @@ object ObsTabContents extends TwoResizablePanels:
           case CopyAlt1 | CopyAlt2 | CopyAlt3 =>
             obs
               .map(id =>
-                ctx.exploreClipboard.set(LocalClipboard.CopiedObservation(id)).runAsync *> info(
-                  s"Copied"
-                )
+                ctx.exploreClipboard
+                  .set(LocalClipboard.CopiedObservations(ObsIdSet.one(id)))
+                  .runAsync *>
+                  info(s"Copied obs $id")
```

Comment:
Do we want to leave the `info` message? Maybe with `debug` level?

Reply:
This is a toast actually, it shows the info that an obs was copied to the user